services/ui-src/src/components/forms/FormLayoutElements.test.tsx

```diff
@@ -0,0 +1,75 @@
+import { render, screen } from "@testing-library/react";
+import { axe } from "jest-axe";
+import { SectionHeader } from "./FormLayoutElements";
+
+const sectionHeaderComponentTopDivider = (
+  <SectionHeader
+    divider="top"
+    name="sectionHeader"
+    content="1. Section Header"
+    data-testid="test-section-header"
+  />
+);
+
+const sectionHeaderComponentBottomDivider = (
+  <SectionHeader
+    divider="bottom"
+    name="sectionHeader"
+    content="1. Section Header"
+    data-testid="test-section-header"
+  />
+);
+
+const sectionHeaderComponentNoDivider = (
+  <SectionHeader
+    divider="none"
+    name="sectionHeader"
+    content="1. Section Header"
+    data-testid="test-section-header"
+  />
+);
+
+describe("Test SectionHeader component", () => {
+  test("Top should make the section divider on the top.", async () => {
+    const { findByText, getByRole } = render(sectionHeaderComponentTopDivider);
+    const sectionHeader = screen.getByTestId("test-section-header");
```

Comment:
we are trying to only use getByTestId when absolutely necessary going forward. I have some suggestions on how to test this without test ids and child nodes

Reply:
I have the code written locally but would like to review with you prior to committing.

Reply:
I am going to need to be done soon to make dinner, if you look at this again before tomorrow:

> describe("Test SectionHeader component", () => {
>   test("Top should make the section divider on the top.", async () => {
>     const { getByRole } = render(sectionHeaderComponentTopDivider);
> 
>     const heading = getByRole("heading", { name: "1. Section Header" });
>     const separator = getByRole("separator");
> 
>     expect(heading).toBeVisible();
>     expect(separator).toBeVisible();
>     expect(heading.compareDocumentPosition(separator)).toBe(
>       Node.DOCUMENT_POSITION_PRECEDING
>     );
>   });

Reply:
This makes sense. Thanks for finding this for me!

Reply:
Fixed!